projects/igniteui-angular/src/lib/grids/filtering/excel-style/grid.excel-style-filtering.component.ts

```diff
@@ -557,7 +559,14 @@ export class IgxGridExcelStyleFilteringComponent implements OnDestroy {
             this.addItems(shouldUpdateSelection);
         }
 
-        this.listData.sort((a, b) => this.sortData(a, b));
+        const sorting = new IgxSorting();
+        const expressions = [{
+            dir: SortingDirection.Asc,
+            fieldName: 'value',
+            ignoreCase: this.column.filteringIgnoreCase,
+            strategy: this.column.sortStrategy
+        }];
+        this.listData = sorting.sort(this.listData, expressions);
```

Comment:
1. We could use the sort method of the strategy directly:
```ts
this.column.sortStrategy.sort(this.listData, 'value', SortingDirection.Asc, this.column.sortingIgnoreCase, ...
```
2. We should use the `sortingIgnoreCase` instead of the `filteringIgnoreCase`.
3. We should handle the `time` column values by extracting the time part from the dates.